ethereum/spec/src/main/java/tech/pegasys/teku/spec/Spec.java

```diff
@@ -884,7 +897,10 @@ public Optional<Integer> getMaxBlobsPerBlock(final UInt64 slot) {
   }
 
   public UInt64 computeSubnetForBlobSidecar(final SignedBlobSidecar signedBlobSidecar) {
-    return signedBlobSidecar.getBlobSidecar().getIndex().mod(BLOB_SIDECAR_SUBNET_COUNT);
+    return signedBlobSidecar
+        .getBlobSidecar()
+        .getIndex()
+        .mod(getNetworkingConfigDeneb().orElseThrow().getBlobSidecarSubnetCount());
```

Comment:
ahh.. here we are.
So, an idea could be to have  `getBlobSidecarSubnetCount` on the base `SpecConfig` and do the override-like game in SpecConfig.

Reply:
Bu I don't actually know how the transition would be handled in our code... so maybe we should do something when we will have to